### ISSUE TITLE
TODO: Add title

### ISSUE THREAD

**Issue by jayantsingh12489** [koalaman/shellcheck#1628]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2003, SC2003
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
echo -e "Please Enter one Value: \c"
read -r c
i=1
while [ $i -le 10 ]
do
b=$(expr "($c)" \* $i)
echo "$c * $i = $b"
i=$(expr $i+1)
done


```

#### Here's what shellcheck currently says:
```
Line 7	SC2003: expr is antiquated. Consider rewriting this using $((..)), ${} or [[ ]].
Line 9	SC2003: expr is antiquated. Consider rewriting this using $((..)), ${} or [[ ]].
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by matthewpersico** [comment]
So what did you expect to see?

**Comment by koalaman** [comment]
Looks good to me. If there's a bug, please reopen with more information.
